static/templates/message_edit_form.hbs

```diff
@@ -68,6 +68,11 @@
                     <a role="button" tabindex=0 class="message-control-button fa fa-video-camera video_link" aria-label="{{t "Add video call" }}" data-message-id="{{message_id}}" title="{{t "Add video call" }}"></a>
                     {{/if}}
                     <a role="button" tabindex=0 class="message-control-button fa fa-smile-o" aria-label="{{t 'Add emoji' }}" id="emoji_map" data-message-id="{{message_id}}" title="{{t 'Add emoji' }}"></a>
+                    {{#if giphy_api_available }}
+                    <a role="button" class="message-control-button" aria-label="{{t 'Add GIF' }}" id="compose_box_giphy_grid" title="{{t 'Add GIF' }}">
+                        <img class="compose_giphy_logo" tabindex=0 src="/static/images/GIPHY_logo.png" data-message-id="{{message_id}}">
+                    </a>
+                    {{/if}}
```

Comment:
The vertical alignment here is off, and I'm not sure why.

![image](https://user-images.githubusercontent.com/2746074/114069198-94239700-9853-11eb-8980-35bc1ec7d6f2.png)

Probably fine for you to just do fix this in a separate PR if we've merged this.  I also think we may want to do TippyJS for all of the titles on all of the `message_control_button` objects both in compose and here.

Reply:
Fixed this.